DOCSP-31387 source database ref page (#40)
mongodb/docs-relational-migrator: source/supported-databases.txt

.. _supported-databases:

=======================================
Supported Source Databases and Versions
=======================================

.. default-domain:: mongodb

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1
   :class: singlecol

Relational Migrator supports migration from source databases to MongoDB. 

Definition
----------

To create a Relational Migrator project, you can connect to a source database to migrate
to MongoDB. This page provides the supported source databases. 

> .. note::

Reviewer: [non-blocking suggestion] I feel like this could be a warning admonition instead of a note. Thoughts?

Author: Yes, thank you!

>    It may be possible to use other versions or deployments but they have not been tested 

Reviewer: [suggestion][ Active voice ](https://www.mongodb.com/docs/meta/style-guide/writing/use-active-voice/)(making user the subject) 

```suggestion
   You might be able to use other versions or deployments for your source database, but they have not been tested 
```

Author: Done.
